coin/coins.go

```diff
@@ -578,6 +580,16 @@ var Coins = map[uint]Coin{
 		MinConfirmations: 0,
 		SampleAddr:       "erd12tqtt5zcg6vpw65y4hkanvt49kzq695sr3ctuszjy92xw0ppzcssy2xd5r",
 	},
+	BSC: {
+		ID:               10000714,
+		Handle:           "bsc",
```

Comment:
binancesmartchain?

Reply:
this one is defined as `bsc` in wallet core, suggests by Viktor